### ISSUE TITLE
Feature request: View settings per view

### ISSUE THREAD

**Issue by fml2** [notepad-plus-plus/notepad-plus-plus#3839]
It would be very helpful if NP++ would allow to set the view settings (e.g. "word wrap", "show all characters" etc.) individually for each view (and not have one global setting affecting all views).

**Comment by sasumner** [comment]
Do you really mean "view"?  Of which there are exactly 2 (view0 and view1), create by the commands "Move (or Clone) to Other View".  Or do you mean these settings should be on a per-document (aka per-tab) basis?

**Comment by fml2** [comment]
Yes, I must correct myself. I updated the title and the description.

**Comment by PeJeyDimO** [comment]
Supported.

There should be an option to set word-wrap per tab/document all the time.
Save the specific setting in session files too.

There is another open issue suggesting it:
https://github.com/notepad-plus-plus/notepad-plus-plus/issues/1427
